cmd/algocfg/profileCommand.go

```diff
@@ -19,39 +19,81 @@ package main
 import (
 	"bufio"
 	"fmt"
-	"github.com/spf13/cobra"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"github.com/spf13/cobra"
+
 	"github.com/algorand/go-algorand/cmd/util/datadir"
 	"github.com/algorand/go-algorand/config"
 	"github.com/algorand/go-algorand/util/codecs"
 )
 
-// profileConfigUpdater updates the provided config for non-defaults in a given profile
-type profileConfigUpdater func(cfg config.Local) config.Local
-
-// defaultConfigUpdater leaves all default values in place
-func defaultConfigUpdater(cfg config.Local) config.Local {
-	return cfg
-}
-
-// relayConfigUpdater alters config values to set up a relay node
-func relayConfigUpdater(cfg config.Local) config.Local {
-	cfg.Archival = true
-	cfg.EnableLedgerService = true
-	cfg.EnableBlockService = true
-	cfg.NetAddress = "4160"
-	return cfg
+// configUpdater updates the provided config for non-defaults in a given profile
+type configUpdater struct {
+	updateFunc  func(cfg config.Local) config.Local
+	description string
 }
 
 var (
+	archival = configUpdater{
+		description: "Store all historic blocks for the API.",
+		updateFunc: func(cfg config.Local) config.Local {
+			cfg.Archival = true
+			cfg.CatchpointTracking = -1
+			return cfg
+		},
+	}
+
+	development = configUpdater{
+		description: "Features useful for building on Algorand.",
+		updateFunc: func(cfg config.Local) config.Local {
+			cfg.EnableExperimentalAPI = true
+			cfg.EnableDeveloperAPI = true
+			cfg.IncomingConnectionsLimit = 0
+			return cfg
+		},
+	}
+
+	follower = configUpdater{
+		description: "A node which can be synchronized with an external application.",
+		updateFunc: func(cfg config.Local) config.Local {
+			cfg.EnableFollowMode = true
+			cfg.MaxAcctLookback = 16
+			cfg.CatchupParallelBlocks = 16
```

Comment:
In our perf tests, I've been using the following:
```
cfg.CatchupParallelBlocks = 64
cfg.EnableFollowMode = true
cfg.MaxAcctLookback = 64
```

Also worth noting that the test script sets
```
cfg. CatchupBlockValidateMode = 3
``` 
which will probably speed up catchup, but has some trust implications obviously.